libs/expo/betterangels/src/lib/screens/Client/Profile/PersonalInfo.tsx

```diff
@@ -38,9 +38,19 @@ export default function PersonalInfo(props: IProfileSectionProps) {
     client?.clientProfile.user.lastName ?? ''
   }`.trim();
 
-  const formattedDob = client?.clientProfile.dateOfBirth
-    ? format(client?.clientProfile.dateOfBirth, 'MM/dd/yyyy')
-    : null;
+  const clientDateOfBirth = client?.clientProfile.dateOfBirth;
```

Comment:
what makes this preferable to the check that's in place now?

Reply:
the value can be truthy and yet still an Invalid Date

Reply:
@mikefeldberg  so I opened this PR as apparently I was working with code that didn't have your  PR #678 (Add null check before formatting date of birth) pulled in. 
I thought I had the most recent code, as this branch visibly has the code from 678, but guess I did not.

So if `client?.clientProfile.dateOfBirth` is guaranteed to be either valid `date` or `null`, which looks like it, then I'm good to close this PR.